cmd/server-main.go

```diff
@@ -413,7 +413,7 @@ func initServer(ctx context.Context, newObject ObjectLayer) error {
 	t1 := time.Now()
 
 	// Once the config is fully loaded, initialize the new object layer.
-	setObjectLayer(newObject)
+	defer setObjectLayer(newObject)
```

Comment:
looks like the mint tests are failing due to this 

```
{
  "alert": "",
  "args": {
    "bucketName": "aws-sdk-go-test-xuin4gr4m14f0l",
    "expiry": 60000000000,
    "objectName": "presignedTest"
  },
  "duration": 364831,
  "error": "XMinioServerNotInitialized: Server not initialized, please try again.\n\tstatus code: 503, request id: 175FCA4CA107CB03, host id: 87ef3db1a67a60b69bcfee10c0874f8a253fc9bc1bf4e58d7314742c975c4eb6",
  "function": "PresignedPut",
  "message": "AWS SDK Go CreateBucket Failed",
  "name": "aws-sdk-go",
  "status": "FAIL"
}
```

Do we need to fix the tests? or add the API freeze to handler?